WordPress/src/main/java/org/wordpress/android/ui/selfhostedusers/SelfHostedUsersScreen.kt

```diff
@@ -188,6 +189,7 @@ private fun UserDetail(
             val avatarUrl = user.avatarUrls?.values?.firstOrNull()
             SmallAvatar(
                 avatarUrl = avatarUrl,
+                contentDescription = stringResource(R.string.user_avatar_content_description, user.name),
```

Comment:
That `contentDescription` is only set  when the user has an actual avatar, because in that situation clicking the avatar shows it full screen. You can test this by setting `avatarUrls` for one of the sample users, like this:

```
avatarUrls = mapOf("sampleUserTwo" to "https://nickbradbury.com/wp-content/uploads/2022/03/1394-2.jpg"),
```